### ISSUE TITLE
TypeError in _getString() due to change in Node.js Buffer.concat()

### ISSUE THREAD

**Issue by kixxauth** [diachedelic/mock-res#5]
Stack trace:

```
Failures:
    TypeError: "list" argument must be an Array of Buffers
        at Function.Buffer.concat (buffer.js:313:11)
        at MockExpressResponse.MockServerResponse._getString (/home/travis/build/oddnetworks/oddworks/node_modules/mock-res/index.js:55:16)
        at MockExpressResponse.MockServerResponse._getJSON (/home/travis/build/oddnetworks/oddworks/node_modules/mock-res/index.js:59:25)
```

Not sure exactly what changed, but I'm going to monkey patch this locally and see what happens.

**Comment by justmoon** [comment]
Submitted a PR to fix this: #6

**Comment by diachedelic** [comment]
Fixed in v0.3.1
